client/my-sites/github-deployments/use-github-installations-query.ts

```diff
@@ -17,6 +17,7 @@ export interface GitHubInstallationData {
 	external_id: number;
 	account_name: string;
 	management_url: string;
+	repository_selection: string;
```

Comment:
We can use constant values since we know what to expect from the API.

```suggestion
	repository_selection: 'all' | 'selected';
```